mod-invoice-storage/examples/invoice_line.sample

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "e0d08448-343b-118a-8c2f-4fb50248d672",
+  "description": "Some description",
+  "poLineId": [
+    "c0d08448-347b-418a-8c2f-5fb50248d67e"
+  ],
+  "invoiceLineNumber": "123invoicenumber45-1",
+  "invoiceLineStatus": "Open",
+  "vendorRefNo": "1",
+  "productId": "9780764354113",
+  "productIdType": "ISBN",
+  "invoiceLineTax": 11.2,
+  "comment": "Sample invoice line",
+  "price": 2.2,
+  "quantity": 3.0,
+  "doNotReleaseEncumbrance": true,
+  "reportTax": 3.5,
+  "subscriptionInfo": "Subscription information",
+  "subscriptionStart": "2018-08-01T00:00:00.000+0000",
+  "subscriptionEnd": "2019-01-01T00:00:00.000+0000",
+  "pieceId": "5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7",
```

Comment:
I think this should be an array (is repeatable).   Also, lets make it plural `pieceIds`  I'll ask Dennis to update the spreadsheet

Reply:
Updated